### ISSUE TITLE
Solr connection does not support TLS

### ISSUE THREAD

**Issue by 4censord** [alexmshepherd/owdex#19]
In https://github.com/alexmshepherd/owdex/blob/f6300e40b4b1402bfe70bd0a9c7447aa160dbcfb/frontend/owdex/solr.py#L7-L10 , the protocol is hardcoded to http. Also, the port is not selectable.

This is more of later sometime thing though.

**Comment by alexmshepherd** [comment]
Yep, it's hardcoded right now. On the current roadplan is #20 which would probably include config for this.

**Comment by alexmshepherd** [comment]
TLS isn't a _huge_ priority, as owdex is intended (at least initially) to have frontend and backend working on the same system in connected Docker containers where the connection won't be exposed to the world. However, it's still a good best practice.
